website/docs/addon_applications_admin.md

```diff
@@ -1,26 +1,235 @@
 
+<ReactMarkdown>
+{versions.Applications_Badge}
+</ReactMarkdown>
 
-## Applications
+## Introduction
 
-In this section you can manage what Applications are available to your studio, locations of their executables, and their additional environments. In AYON context, each application that is integrated is also called a `Host` and these two terms will be used interchangeably in the documentation.
+The Applications addon is designed to configure applications, set their executable paths, and set up their startup environments.
 
-Each Host is made of two levels.
-1. **Application group** - This is the main name of the application and you can define extra environments that are applicable to all versions of the given application. For example any extra Maya scripts that are not
-version dependent, can be added to `Maya` environment here.
-2. **Application versions** - Here you can define executables (per platform) for each supported version of
-the DCC and any default arguments (`--nukex` for instance). You can also further extend its environment.
+<!-- :::info
+The addon introduces a debug launcher action for users. For more details, please refer to [Application addon artist docs] (addon_applications_artist.md)
+Also, The addon's client code features hook definitions that developers can leverage to implement launch hooks that tailor launch processes for other addons. For more details, please see [Application addon Dev docs] (addon_applications_dev.md).
+::: -->
 
-![settings_applications](assets/settings/applications_01.png)
+![settings_applications](assets/applications/admin_applications_layers.png)
 
-### Environments
+The Addon features two primary settings categories: Definitions and Filters:
+
+1. **Definitions:** available only in studio settings, allows to define all possible applications and tools that can be used across all projects.
+2. **Filters:** defines when certain applications and tools are used on project level. This filters whether a certain application or tool should be enabled in a certain launch context, like a specific task type or project-wide.
+
+## Applications Addon Settings
+### Show only available applications
+![](assets/applications/show_only_available_apps.png)
+
+When enabled, AYON Launcher will display only those applications for which executable paths exist on the current machine.
+This acts as a extra filter to the Applications filters, ensuring that inaccessible applications are excluded from view.
+
+### Applications Definitions
+
+In this section you can manage what Applications are available to your studio, locations of their executables, and their additional environments. 
+In AYON context, each application that is integrated is also called a `Host` and these two terms will be used interchangeably in the documentation.
+
+#### Built-in Applications
+Each integrated application (`Host`) has an application group e.g. Maya, Houdini, Nuke and etc.
```

Comment:
I would say that this explanation will create more confusion about app vs. host.

Reply:
```suggestion
Each application definition has an application group e.g. Maya, Houdini, Nuke and etc.
```